### PR TITLE
Add support for Ubuntu 24.04 in BV

### PR DESCRIPTION
**Pull request by Bischoff**
This PR adds support for Ubuntu 24.04. Part of SUSE/spacewalk#23851.

Continuation of #1354 (Dominik absent)

Note: there is no ubuntu 24.04 support on 4.3.

### CODE REVIEW

File: jenkins_pipelines/scripts/json_generator/maintenance_json_generator.py [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "alma8_minion": {"/SUSE_Updates_RES_8-CLIENT-TOOLS_x86_64/"},
     "ubuntu2004_minion": {"/SUSE_Updates_Ubuntu_20.04-CLIENT-TOOLS_x86_64/"},
     "ubuntu2204_minion": {"/SUSE_Updates_Ubuntu_22.04-CLIENT-TOOLS_x86_64/"},
+    # no Ubuntu 24.04 on 4.3
```
Comment (ktsamis) [inline jenkins_pipelines/scripts/json_generator/maintenance_json_generator.py:66]: we need this for 5.0 as well though, the dictionaries are merged. Or add it in the 5.0 with the non beta one.

**Review by ktsamis — CHANGES_REQUESTED** [review]
The json generator needs adjusting but otherwise great

File: jenkins_pipelines/scripts/json_generator/maintenance_json_generator.py (hunk 1 above)
Comment (Bischoff) [reply jenkins_pipelines/scripts/json_generator/maintenance_json_generator.py:66]: i added to the 5.0 section.

For the moment the tools are not published to SCC, maybe my addition to 5.0 section was wrong?

**Review by ktsamis — APPROVED** [review]
All good now, thanks a lot for fixing this

**Comment by NamelessOne91** [comment]
Thanks !

**Pull request merged**